apps/percona/chaos/openebs_pool_network/test.yml

```diff
@@ -53,16 +34,15 @@
         ## PRE-CHAOS APPLICATION LIVENESS CHECK
 
         - name: Verify that the AUT (Application Under Test) is running 
-          shell: >
-            kubectl get pods -n {{ namespace }} -l {{ label }} --no-headers
-            -o custom-columns=:status.phase
-          args:
-            executable: /bin/bash
-          register: app_status
-          until: "((app_status.stdout_lines|unique)|length) == 1 and 'Running' in app_status.stdout"
-          delay: 10
-          retries: 12
-
+          include_tasks: /common/utils/status_app_pod.yml
+          vars:
+            application_name: "Percona"
```

Comment:
@vibhor995  can you check this application_name is really need here?

Reply:
@nsathyaseelan we can pass it as a env if needed.